lib/blockchain/cardano.js

```diff
@@ -0,0 +1,47 @@
+const fs = require('fs')
+const path = require('path')
+const common = require('./common')
+const mnemonicHelpers = require('../../lib/mnemonic-helpers')
+
+const coinUtils = require('../coin-utils')
+const options = require('../options')
+
+module.exports = { setup }
+
+const es = common.es
+
+// todo: maybe add '#' as second parameter?
+function escapeRegExp(str) {
```

Comment:
it's used below in `setup` regexp for change default folder, but in this regexp we use `#`, so i build own escape function instead of using `_.escapeRegExp`

Reply:
If the only difference compared to lodash is that we need to escape extra `#`, what about calling `_.escapeRegExp` and we will do only escape of '#' on the result of it? It might be safer than enumerating all escape characters ourselves.
Something like:
```
let res = _.escapeRegExp(str)
// Note: we also need to escape '#'
return res.replace(/#/g, '\\$&')
```